docs: hepa door rollers 2 (#19370)
OpenTrons/opentrons-api: docs/hepa-uv/docs/product-specs.md

- Typical power consumption: 75.5 W
- Peak power consumption: 160 W

## Door Protection Rollers

> With early model HEPA/UV modules, fully opening the robot's front door meant scraping it across the top of that module. Eventually, the edge of the HEPA module would scratch and leave scuff marks on the surface of the door.

Reviewer: Even though I think this was happening a lot, let's soften our modal verbs:
```suggestion
With early model HEPA/UV Modules, fully opening the robot's front door could cause it to scrape against the top of the module. The edge of the HEPA/UV Module could scratch and leave scuff marks on the surface of the door.
```

> ![Module without door rollers](images/hepa-without-rollers.png)

Reviewer: Optional: move this after the next paragraph and do a `<figure class="side-by-side">` for a nice visual comparison.

Author: Yes, let's make it side by side.

> With early model HEPA/UV Modules, fully opening the robot's front door could cause it to scrape against the top of the module. HEPA modules manufactured on or after July 2025 include three rollers mounted along its top-front edge. These rollers help prevent the HEPA module from damaging the front door when you open it.

Reviewer: Tiny changes. We have to remember the `/UV` because the HEPA module (no UV) is a different product entirely.

```suggestion
With early model HEPA/UV Modules, fully opening the robot's front door could cause it to scrape against the top of the module. HEPA/UV Modules manufactured in or after July 2025 include three rollers mounted along its top-front edge. These rollers help prevent the HEPA/UV Module from damaging the front door when you open it.
```

## Door Protection Rollers

With early model HEPA/UV Modules, fully opening the robot's front door could cause it to scrape against the top of the module. HEPA modules manufactured on or after July 2025 include three rollers mounted along its top-front edge. These rollers help prevent the HEPA module from damaging the front door when you open it.

> ![Modules without and with rollers](images/hepa-comparison.png)

Reviewer: Ah, so I was actually thinking of doing something like this:

```
<figure class="side-by-side" markdown>
![Module with door rollers](images/hepa-with-rollers.png)
![Module without door rollers](images/hepa-without-rollers.png)
<figcaption>Newer models have door rollers. Older models do not.</figcaption>
</figure>
```

Author: That's good too. I'm assuming this is a class rule that puts 2 separate images side by side. Will do that.
